setup.py

```diff
@@ -57,15 +57,17 @@ def find_long_description() -> str:
     install_requires=[  # Make sure to keep in sync with Pipfile requirements.
         "arrow>=0.15.0,<1",
         "funcy>=1.7.3,<2",
-        "graphql-core>=3.1,<3.2",
+        "graphql-core>=3.1.2,<3.2",
```

Comment:
Just getting this in line with our `Pipfile` — since this is a patch release, I don't think it's a big deal.